src/tespy/networks/networks.py

```diff
@@ -2511,6 +2511,134 @@ def postprocessing(self):
+
+                E_{\text{P}} = P_{\text{net}}\\
+                E_{\text{F}} = \sum_{comp=0}^N E_{\text{F},comp}\\
+                E_{\text{L}} = \sum_{conn=0}^N E_{\text{PH},conn}\\
+                E_{\text{D}} = E_{\text{F}} - E_{\text{P}} - E_{\text{L}}\\
+                \varepsilon = \frac{E_{\text{P}}}{E_{\text{F}}}
+
+                y_{\text{D},comp} =
+                \frac{\dot{E}_{\text{D},comp}}{\dot{E}_{\text{F}}}\\
+                y^*_{\text{D},comp} =
+                \frac{\dot{E}_{\text{D},comp}}{\dot{E}_{\text{D}}}
+        """
+        if E_F is None:
+            msg = ('Missing fuel exergy E_F of network.')
+            logging.warning(msg)
+
+        # physical exergy of connections
+        for conn in self.conns.index:
+            conn.get_physical_exergy(pamb, Tamb)
+
+        # exergy balance of components
+        for cp in self.comps.index:
+            if (isinstance(cp, cycle_closer) or isinstance(cp, sink) or
+                    isinstance(cp, source) or isinstance(cp, splitter)):
+                continue
+            cp.exergy_balance(Tamb, bus)
+
+            # calculate E_D for components
+            try:
+                if isinstance(cp.E_P, str):
+                    cp.E_D = cp.E_F
+                else:
+                    cp.E_D = cp.E_F - cp.E_P
+            except AttributeError:
+                cp.E_D = 0
+                cp.E_F = 0
+                cp.E_P = 0
+            # calculate epsilon for components
+            try:
+                if isinstance(cp.E_P, str):
+                    cp.epsilon = 'n/a'
+                else:
+                    cp.epsilon = cp.E_P / cp.E_F
+            except ZeroDivisionError:
+                cp.epsilon = -111
+
+        # bottom up calculation of exergy destruction
+        self.E_D_sum = 0
+        for cp in self.comps.index:
+            if (isinstance(cp, cycle_closer) or isinstance(cp, sink) or
+                    isinstance(cp, source) or isinstance(cp, splitter)):
+                continue
+            if cp.epsilon == -111:
+                continue
+            self.E_D_sum += cp.E_D
+
+        # calculate E_P, E_F, E_L, E_D and epsilon for network
+        self.E_L = 0
+        self.E_F = 0
+
+        for b in self.busses.values():
+            self.E_P = b.P.val*-1
+
+        for cp in E_F:
+            self.E_F += cp.E_F
+
+        if E_L is not None:
+            for conn in E_L:
+                self.E_L += conn.Ex_Physical
+
+        self.E_D = self.E_F - self.E_P - self.E_L
+        self.epsilon = self.E_P/self.E_F
+
+        # calculate exergy destruction ratios for components
+        for cp in self.comps.index:
+            if (isinstance(cp, cycle_closer) or isinstance(cp, sink) or
+                    isinstance(cp, source) or isinstance(cp, splitter)):
+                continue
+            if cp.epsilon == -111:
+                continue
+            cp.y_Dk = cp.E_D / self.E_F
+            cp.ystar_Dk = cp.E_D / self.E_D
```

Comment:
On top of the equations we need to have a detailed description of the parameters we calculate, so it is clear what each of the ratios and numbers mean.

Reply:
Alright, I will describe it in more detail

Reply:
We'll also need a literature reference for the notation.